.github/workflows/lint.yml

```diff
@@ -24,8 +25,12 @@ jobs:
           chmod +x llvm.sh
           sudo ./llvm.sh 18
           sudo apt-get install -y --no-install-recommends clang-format-18
+          curl -fsSL https://deb.nodesource.com/setup_22.x | sudo bash
+          sudo apt-get install -y nodejs
+          npm i -g prettier
```

Comment:
We're not defining a version of prettier here – I assume that means we'll be using the latest stable version and CI will break without warning whenever there's a prettier update that adds new formatting rules/fixes bugs with existing ones that affect our code base. Can we define a fixed version here?
A related issue is that locally we will run whatever prettier version is available locally – if it disagrees with the prettier version on CI developers won't be able to contribute code until they upload their version. Perhaps we can use the existing bazel facilities to run prettier? I noticed that prettier is already listed in package.json, although I'm not sure how it is used so far. Perhaps it could work kind of like the `//src/cloudflare:cloudflare.capnp@eslint` and `//src/node:node.capnp@eslint` bazel targets we have for eslint?

Reply:
I think we should just run `pnpm install` and later execute the prettier using `pnpm lint` etc. To reduce the friction of maintaining these kinds of things.

Reply:
I chose the pnpm install approach in order to keep the `lint` Github action fast and simple.